bsoid_app/extract_features.py

```diff
@@ -177,7 +177,7 @@ def main(self):
             st.markdown('CHECK POINT: Done non-linear transformation of {} instances '
                         'from {} D into {} D. Move on to Identify and '
                         'tweak number of clusters'.format(*self.sampled_features.shape, self.sampled_embeddings.shape[1]))
-            caching.clear_cache()
+            #caching.clear_cache()
```

Comment:
why comment this out? should it be fully removed?

Reply:
put back